### PR TITLE
Add descriptions for new options for JSON parser plugin.

### PR DESCRIPTION
**Pull request by kamatama41**
For #1102 
As we fixed option names in #1120, it's time to add them into the official doc.

### CODE REVIEW

File: embulk-docs/src/built-in.rst [hunk 1]
```diff
@@ -353,13 +353,25 @@ The ``json`` parser plugin parses a JSON file that contains a sequence of JSON o
...
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| flatten_json_array         | boolean  | Set true when regard elements in a JSON array as multiple Embulk records.                                       | ``false`` by default                   |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| columns                    | hash     | Columns (see below)                                                                                             | optional                               |
++----------------------------+----------+-----------------------------------------------------------------------------------------------------------------+----------------------------------------+
+| default\_timezone          | string   | Time zone of timestamp columns. This can be overwritten for each column using ``column``                        | ``UTC`` by default                     |
```
Comment (dmikurube) [inline embulk-docs/src/built-in.rst]: Yeah, `default_timezone`, `default_timestamp_format`, and `default_date` can be configured actually, but in a rough direction, we're going to deprecate them.

Sorry that I didn't let you know, but let's keep them undocumented.

File: embulk-docs/src/built-in.rst [hunk 2]
```diff
@@ -376,6 +388,23 @@ if you set invalid\_string\_escapes and appear invalid JSON string (such as ``\a
 
 (\*1): Throwing an exception.
 
+The ``columns`` option can extract any value as an Embulk's column.
```
Comment (dmikurube) [inline embulk-docs/src/built-in.rst]: How about :
```
The ``columns`` option declares the list of columns, and the way how to extract JSON values into Embulk columns.
```

File: embulk-docs/src/built-in.rst [hunk 3]
```diff
@@ -376,6 +388,23 @@ if you set invalid\_string\_escapes and appear invalid JSON string (such as ``\a
...
+
++----------+-----------------------------------------------------------------------------------------------------------+
+| name     | description                                                                                               |
++==========+===========================================================================================================+
+| name     | Name of the column. Select a key within a JSON record or any name can be allowed with ``pointer`` option. |
```
Comment (dmikurube) [inline embulk-docs/src/built-in.rst]:
```
Name of the column. The JSON value with this name is extracted if `pointer` is not specified.
```

File: embulk-docs/src/built-in.rst [hunk 4]
```diff
@@ -376,6 +388,23 @@ if you set invalid\_string\_escapes and appear invalid JSON string (such as ``\a
...
++----------+-----------------------------------------------------------------------------------------------------------+
+| type     | Type of the column (same as CSV parser's one)                                                             |
++----------+-----------------------------------------------------------------------------------------------------------+
+| pointer  | Set as a JSON Pointer expression if you want to specify any value in a JSON record as the column's value  |
++----------+-----------------------------------------------------------------------------------------------------------+
+| timezone | Set if you want to specify time zone for the timestamp column                                             |
```
Comment (dmikurube) [inline embulk-docs/src/built-in.rst]: For a similar reason with above, let's keep only `format` here without `timezone` and `date`.

File: embulk-docs/src/built-in.rst [hunk 5]
```diff
@@ -376,6 +388,23 @@ if you set invalid\_string\_escapes and appear invalid JSON string (such as ``\a
...
+The ``columns`` option can extract any value as an Embulk's column.
+
++----------+-----------------------------------------------------------------------------------------------------------+
+| name     | description                                                                                               |
++==========+===========================================================================================================+
+| name     | Name of the column. Select a key within a JSON record or any name can be allowed with ``pointer`` option. |
++----------+-----------------------------------------------------------------------------------------------------------+
+| type     | Type of the column (same as CSV parser's one)                                                             |
++----------+-----------------------------------------------------------------------------------------------------------+
+| pointer  | Set as a JSON Pointer expression if you want to specify any value in a JSON record as the column's value  |
```
Comment (dmikurube) [inline embulk-docs/src/built-in.rst]: How about :
```
Pointer to the child node to be extracted as the column, expressed as a JSON Pointer (optional).
```

Comment (dmikurube) [reply embulk-docs/src/built-in.rst]: Ah, it may not be `child`, but should be `descendant`.

**Comment by kamatama41** [comment]
@dmikurube 
Thank you for your suggestion. I have updated following them in ef3c2fa

**Pull request merged**